### DISCUSSION TITLE
m2eclipse artworks?

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-m2e/m2e-core#912] [Q&A]
I noted that we don't use an icon for the m2e org, and also probably like to use some icon/logo in my eclipse webinar/eclipsecon talk. So I wonder if there is any Artwork available, the only thing I could find by poking around is:

![](https://github.com/eclipse-m2e/m2e-core/raw/master/org.eclipse.m2e.core.ui/icons/m2eclipse48.gif)

what seems a bit low-resolution... but is what one also finds in the IDE.

The [m2e website](https://www.eclipse.org/m2e/) on the other hand seem to use 

![](https://www.eclipse.org/m2e/assets/img/m2e_logo.png)

but I never seen this anywhere else.

So is there some kind of "official" logo and corresponding files we can use to promote m2eclipse?

**Comment by HannesWell** [comment]
> So is there some kind of "official" logo and corresponding files we can use to promote m2eclipse?

I'm not aware of more than you have already mentioned.

But maybe this is a good opportunity to (re-)create a large one?

In general I like the first icon because it refers to Eclipse and Maven which M2E connects. But it could use a more recent Eclipse-Icon/logo base/background? And maybe the `2` in `m2` could be dropped and just an `M` could be used? IIRC I have seen some kind of Logo/Branding writing `Maven` in the same red from Maven, but know when looking at the [Maven Website](https://maven.apache.org/) I can only see the/a feather/s as logo. Maybe the red `Maven` writing was an old logo of Maven?

I don't understand the meaning of the logo on the website? Are the two 'mountains' are supposed to form an `M`?

**Reply by mickaelistria** [reply]
The one that looks like 2 mountains hiding a bit of the moon is the more official one I believe, as it's the one on the website and on the marketplace entry https://marketplace.eclipse.org/content/eclipse-m2e-maven-support-eclipse-ide .

A piece of advice from my former experience on that matter (use it or not, as you like it or not ;) is to not waste time on reworking the branding if you don't have a clear idea of the benefits you get from it. m2e is not really worth branding effort, m2e is for users the Maven support of the Eclipse IDE and it's better than way; users of m2e won't be better served by a stronger branding, they would actually be annoyed to see more visible presence of "m2e" (the name, the logo...) in the IDE, they want "Maven" and that's all. Also such a stronger branding actually takes a non-neglectable time to elaborate, setup and deploy. The ROI of such branding effort is IMO very low for an internal component of the Eclipse IDE.

**Reply by laeubi** [reply]
I think the "m2" might also reference to maven2 ... so probably as of today we more would new a m3 logo ;-)

Anyways id the one from the website is the "official one" I created: https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1942 and will probably use that in talks when referencing m2eclipse.

Beside that I think branding actually **do** matter as people recognize it and better can associate it with the work done under the projects umbrella.  Still improving/recreating things here is a bit out of my scope currently.

By the way the way it would be good to have the website "news" updates it still missing the new 2.x release: https://www.eclipse.org/m2e/m2e-news.html

**Comment by laeubi** [comment]
I noticed the m2e website is gone? Anyone knows what happening?

https://www.eclipse.org/m2e/ just redirects to the eclipse project page... I mostly miss the the "how to make a plugin compatible" page...

**Reply by HannesWell** [reply]
No idea, but that is not good.
The content of the website is probably still available in https://git.eclipse.org/c/www-deleted/m2e.git/

From the name of that repo it looks like this was intentional but I'm not aware of an announcement that project websites are about to be deleted.
@laeubi do you think it would be reasonable to create a help-desk issue to at least find out what happened?
Eventually there should/will probably another site, similar like in Tycho that the old website is redirected to.

**Reply by laeubi** [reply]
we can setup a site for m2e in github of course. Or we can use the wiki for that, at laest it was a bit surprsing. Beside that anyone can open HelpDesk tickets but it might be good to decide what we want @fbricon @mickaelistria @akurtakov

**Reply by HannesWell** [reply]
> we can setup a site for m2e in github of course. Or we can use the wiki for that, at laest it was a bit surprsing. Beside that anyone can open HelpDesk tickets but it might be good to decide what we want @fbricon @mickaelistria @akurtakov

Let's continue the discussion at #831.
